### ISSUE TITLE
Python is unfarily short because it uses PIL library.

### ISSUE THREAD

**Issue by treeform** [edin/raytracer#25]
Python appears to be the only language to use external library PIL to write the bitmap out. This makes it significantly shorter then other languages.

I think to be fair to other languages in the lines of code benchmark Python should not use PIL and write the .bmp like the other languages.

Nim's implantation has double empty lines and comments other languages do not have. If Python is forced to write out the bitmap and Nim's code is cleaned up I think Nim might be able to take the top spot in both speed and shortness.

**Comment by edin** [comment]
I agree, ruby I think also uses libraray to write bitmap image, I currently don't have much time but I'll try remove dependency on external libraries for ruby and python.
